crates/rattler_shell/src/shell/mod.rs

```diff
@@ -244,9 +263,12 @@ impl Shell for PowerShell {
     fn extension(&self) -> &str {
         "ps1"
     }
+    fn executable(&self) -> &str {
+        "powershell"
```

Comment:
pwsh[.exe] is the executable file name of PowerShell [Core] (v6+), the cross-platform edition of PowerShell built on .NET Core / .NET 5+; by contrast, powershell.exe is the executable name of the legacy Windows PowerShell edition (v5.1-), built on the Windows-only .NET Framework (v4.8-)

Reply:
Should we use `pwsh`? cc @baszalmstra